Assets/Scripts/Tests/TestCommanderSelectUnit.cs

```diff
@@ -12,13 +12,19 @@ public class TestCommanderSelectUnit
         public void TestSelectUnit()
         {
             Zerg zerg = new Zerg();
-            Lavar lavar = new Lavar();
-            zerg.AddUnit(lavar, zerg.MyUnits);
-            Unit testAddUnit = zerg.MyUnits[0];
-            Assert.AreEqual(lavar, testAddUnit);
-            Assert.AreEqual(false, testAddUnit.Selected);
-            zerg.SelectUnit(testAddUnit);
-            Assert.AreEqual(true, testAddUnit.Selected);
+            // zerg 유닛 만들기
+            zerg.AddUnit(new Lavar());
+            zerg.AddUnit(new Drone());
+
+            // 라바 한마리 가져오기
+            Unit testUnit = zerg.GetUnits("LAVAR")[0];
```

Comment:
@jongfeel 

유닛 선택하기 메서드를 한마리/그룹 으로 나누게 되었는데요! 
그래서 유닛을 가져오는 메서드를 타입별 리스트로 가져오고
한마리씩 가져오는 방법은 인덱스를 사용했는데 요게 알맞은 방법일까요?

Reply:
맞는 방법은 아니에요.

그리고 string value로만 parameter를 주고 있는데 만약 lavar가 두 마리 이상이면 어떤 lavar를 선택해야 하는지에 대한 건 선택한 쪽에서 판단하는게 맞는 것 같습니다.
```suggestion
            Unit testUnit = null;
            List<Unit> lavars = zerg.GetUnits("LAVAR");
            if (lavars != null && lavars.Count > 0)
            {
                testUnit = lavars[0];  // 여기서 두 마리 이상이라면 어떤 lavar를 선택해야 하는지 결정하는 로직도 필요함, index 0 or 1?
            }
```